corehq/blobs/tests/fixtures.py

```diff
@@ -0,0 +1,11 @@
+from unmagic import fixture
+
+from corehq.blobs import get_blob_db
+from corehq.blobs.tests.util import TemporaryFilesystemBlobDB
+
+
+@fixture(scope='function')
```

Comment:
nit: function is the default scope
```suggestion
@fixture
```

Reply:
Addressed in b163a09933a06bf665cb817ed2b67e5300e02699